package.json

```diff
@@ -1 +1,6 @@
-{}
+{
+  "dependencies": {
+    "dgraph-js": "^20.3.1",
```

Comment:
We run DGraph version 20.07.1. It's not clear whether the JS client supports this version from the table here: https://github.com/dgraph-io/dgraph-js#supported-versions

Looking at the releases page, the most recent release is 20.03.0: https://github.com/dgraph-io/dgraph-js/releases

Did you find version 20.3.1 published somewhere? It doesn't seem like it should exist...